app/views/cases/data_requests/_data_request.html.slim

```diff
@@ -0,0 +1,12 @@
+.data-request__single.js-data-request__form data-index="#{form.index}"
+  h3.page-heading--secondary.data-request__position
```

Comment:
Should be h2 for accessibility, I believe - screen reader users expect the headings to be used in sequential order and not miss out levels. But super tiny issue

Reply:
This is a partial included on the `cases/show` page which already has `<h2>` elements. Therefore using `<h3>` here.

Reply:
That's the other partial - this is the form fields in the new data requests form. When you go to `http://localhost:3000/cases/9/data_requests/new`